Semana_20_Testes_Unitarios/TestesUnitarios/TestesUnitarios.Tests/Avaliacao/Business/GuiaFake.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using TestesUnitarios.Avaliacao.Dao;
+using TestesUnitarios.Avaliacao.Entidades;
+
+namespace TestesUnitarios.Tests.Avaliacao.Business
+{
+    public class GuiaFake
+    {
+        private readonly IDao<IGuiaProperties> dao;
+        private IGuiaProperties guia;
+
+        public GuiaFake(IDao<IGuiaProperties> dao)
+        {
+            this.dao = dao;
+            guia = dao.Create();
```

Comment:
👎 Deveria utilizar `Substitute.For<IGuiaProperties>()`, não faz muito sentido precisar de um dao aqui